engine/src/main/java/org/camunda/bpm/engine/impl/bpmn/deployer/BpmnDeployer.java

```diff
@@ -69,6 +73,7 @@
  */
 public class BpmnDeployer extends AbstractDefinitionDeployer<ProcessDefinitionEntity> {
 
+  private static final Logger log = LoggerFactory.getLogger(BpmnDeployer.class);
```

Comment:
:x: The logging in the process engine is done differently. I would avoid introducing this logger.
```suggestion
```